administration/src/cards/extensions/__tests__/StartDayExtension.test.tsx

```diff
@@ -0,0 +1,189 @@
+import { LocalizationProvider } from '@mui/x-date-pickers'
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns'
+import { fireEvent } from '@testing-library/react'
+import React, { ReactNode } from 'react'
+
+import { AppToasterProvider } from '../../../bp-modules/AppToaster'
+import { renderWithTranslation } from '../../../testing/render'
+import PlainDate from '../../../util/PlainDate'
+import { maxCardValidity } from '../../constants'
+import StartDayExtension, { minStartDay } from '../StartDayExtension'
+
+const wrapper = ({ children }: { children: ReactNode }) => (
+  <LocalizationProvider dateAdapter={AdapterDateFns}>
+    <AppToasterProvider>{children}</AppToasterProvider>
+  </LocalizationProvider>
+)
+jest.useFakeTimers({ now: new Date('2024-01-01T00:00:00.000Z') })
+
+describe('StartDayExtension', () => {
+  describe('Component', () => {
+    const mockSetValue = jest.fn()
+    const defaultProps = {
+      value: { startDay: new PlainDate(2024, 1, 1) },
+      setValue: mockSetValue,
+      isValid: true,
+      showRequired: false,
+    }
+
+    it('should render date picker with correct initial value', () => {
+      const { getByDisplayValue } = renderWithTranslation(<StartDayExtension.Component {...defaultProps} />, {
+        wrapper,
+      })
+      expect(getByDisplayValue('01.01.2024')).toBeTruthy()
+    })
+
+    it('should clear input when clear button is clicked', () => {
+      const { getByTitle } = renderWithTranslation(<StartDayExtension.Component {...defaultProps} />, {
+        wrapper,
+      })
+      const clearButton = getByTitle('Wert leeren')
+      fireEvent.click(clearButton)
+      expect(mockSetValue).toHaveBeenCalledWith({ startDay: null })
+    })
+
+    it('should show error message when startDay is empty and field was touched', () => {
+      const { getByText, getByPlaceholderText } = renderWithTranslation(
+        <StartDayExtension.Component {...defaultProps} isValid={false} value={{ startDay: null }} />,
+        { wrapper }
+      )
+      const datePicker = getByPlaceholderText('TT.MM.JJJJ')
+      fireEvent.blur(datePicker)
+      expect(getByText('Bitte geben Sie ein gültiges Startdatum ein.')).toBeTruthy()
+    })
+
+    it('should show error message when startDay is too far in the past and touched', () => {
+      const startDay = new PlainDate(2005, 1, 1)
+      const { getByText, getByDisplayValue } = renderWithTranslation(
+        <StartDayExtension.Component {...defaultProps} isValid={false} value={{ startDay }} />,
+        { wrapper }
+      )
+      const datePicker = getByDisplayValue(startDay.format())
+      fireEvent.blur(datePicker)
+      expect(
+        getByText(`Das Startdatum darf nicht weiter als ${minStartDay.format()} in der Vergangenheit liegen.`)
+      ).toBeTruthy()
+    })
+
+    it('should show error message when startDay is too far in the future and touched', () => {
+      const startDay = new PlainDate(2200, 1, 1)
```

Comment:
same here.
I would recommend to test max start date + 1 as invalid value and max start date as valid value